docs/storage.html

```diff
@@ -250,7 +262,7 @@
         <a href="workflow.html" class="btn btn-neutral float-right" title="Workflows" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
       
       
-        <a href="jobs.html" class="btn btn-neutral float-left" title="Jobs and Job Status" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
+        <a href="review-submissions.html" class="btn btn-neutral float-left" title="Review Submissions" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
```

Comment:
like, in this context, I think Workflow Submissions... you're right  - would be less ambiguous than just `Submissions`

Reply:
I think it's ok to stick with just submissions for now- the application of these classes will be demonstrated in the initial docs under Workflows / Review etc.